### PR TITLE
Fix SpaceModelModules behavior

### PR DESCRIPTION
**Pull request by acs-ferreira**

### CODE REVIEW

**Comment by acs-ferreira** [comment]
#2953 fix CodeClimate issues

File: protected/humhub/modules/space/behaviors/SpaceModelModules.php [hunk 1]
```diff
@@ -23,8 +25,8 @@
 class SpaceModelModules extends Behavior
 {
 
-    public $_enabledModules = null;
-    public $_availableModules = null;
+    public $enabledModules = null;
+    public $availableModules = null;
```
Comment (githubjeka) [inline protected/humhub/modules/space/behaviors/SpaceModelModules.php]: BC break

File: protected/humhub/modules/space/behaviors/SpaceModelModules.php [hunk 2]
```diff
@@ -34,19 +36,21 @@ class SpaceModelModules extends Behavior
     public function getAvailableModules()
     {
 
-        if ($this->_availableModules !== null) {
-            return $this->_availableModules;
+        if ($this->availableModules !== null) {
+            return $this->availableModules;
```
Comment (githubjeka) [inline protected/humhub/modules/space/behaviors/SpaceModelModules.php]: Similar blocks of code found in 2 locations. Consider refactoring 
from https://codeclimate.com/github/humhub/humhub/pull/2951

Comment (acs-ferreira) [reply protected/humhub/modules/space/behaviors/SpaceModelModules.php]: Should be fixed by #2953 as indicated.

Comment (githubjeka) [reply protected/humhub/modules/space/behaviors/SpaceModelModules.php]: Where? `protected/humhub/modules/space/behaviors/SpaceModelModules.php` != ` protected/humhub/modules/user/behaviors/UserModelModules.php`

**Pull request closed without merging**